integration-tests/tasks/tssc-e2e.yaml

```
@@ -0,0 +1,153 @@
+      image: $(params.tssc-test-image)
+      volumeMounts:
+        - name: tssc-test-dir
+          mountPath: /e2e-test
+      script: |
+        #!/usr/bin/env bash
+        set -o errexit
+        set -o nounset
+        set -x
+
+        ls
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)